spec/system/provider_interface/provider_changes_a_course_before_point_of_offer_spec.rb

```diff
@@ -18,6 +18,74 @@
   end
   let(:course_option) { build(:course_option, course: course) }
 
+  # TODO: Uncomment these once location is built
+
+  # scenario 'Changing a course choice before point of offer (no study mode or location)' do
```

Comment:
should we instead only test the full happy path in system spec and the edge-cases in unit tests?